Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -323,6 +315,24 @@ namespace O3DE::ProjectManager
 
             RedirectOutput::Intialize(PyImport_ImportModule("azlmbr_redirect"), &PythonBindings::OnStdOut, &PythonBindings::OnStdError);
 
+            // Add custom site packages after initializing the interpreter above.  Calling Py_SetPath before initialization
+            // alters the behavior of the initializer to not compute default search paths. See
+            // https://docs.python.org/3/c-api/init.html#c.Py_SetPath
+
+            AZ::IO::FixedMaxPath thirdPartyFolder = AzToolsFramework::EmbeddedPython::PythonLoader::GetDefault3rdPartyPath(false);
```

Comment:
This could be a good place to query the user Supplied 3rdParty path by calling [PythonBindings::GetEngineInfo](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/PythonBindings.h#L40) function and then checking the grabbing the [m_thirdPartyPath](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/EngineInfo.h#L29) member off of it.

Reply:
The 3rd Party folder was part of initializing the pythonbindings, so it needs a different approach (see comment above)